### ISSUE TITLE
Add documentation page or section for Migration

### ISSUE THREAD

**Issue by notchris** [payloadcms/payload#2371]
# Documentation Issue

I think it would be helpful to have a page or section on the official docs explaining migration with Payload. Because payload does not run migrations like an ORM would, it is up to the user to apply a migration. There have been issues posted related to migrations and I believe it it would be helpful to users to touch on the subject.

I wrote a quick tutorial on how to setup a basic migration for Payload. The important parts could be editing and used for the docs if easier.

https://gist.github.com/notchris/c5155073bbbb08a487b349c52f6823ae

## Additional Details

Payload rulez

**Comment by jmikrut** [comment]
I totally agree with this. 

We'll get this done!

**Comment by HelenJonathan** [comment]
Hi @jmikrut has this issue been taken care of?
If no, I will like to write the documentation.

**Comment by notchris** [comment]
@HelenJonathan I don't think it has been written yet. There's some notes on the gist I included.

**Comment by DanRibbens** [comment]
Hey @HelenJonathan,

I appreciate your willingness to lend a hand here. 

I'm wondering what you think about this. The reason this hasn't been written yet is because we are actively devloping the database adapter and including support for migrations into that work. I wanted to avoid documenting migrations on the current feature set because it doesn't represent the best DX that Payload is known for.

We're coming close to having it ready for beta testing in the next 2 weeks and fully released in Payload v2.0 before the end of July.

I'm open to ideas around this, but my recommendation would be to just focus on supporting proper Payload migrations and avoid half measures in the documents.

Thoughts?

**Comment by HelenJonathan** [comment]
Hi Dan,

I see the point in what you just said, since a new feature will be released
soon.

There is no point writing documentation at this point. I think it’s best to
wait and do this after the release.

Thanks for the heads up, I appreciate.

**Comment by HelenJonathan** [comment]
Hi Dan,

I am sorry to disturb you.

Last month, I said I was interested in updating an area of your
documentation page, but I wasn't able to continue.
This was due to a new feature that was to be released soon.

It has been over a month, and I am wondering whether that feature has been
released yet, and if I can now go ahead with the documentation.

Yours Sincerely,
Helen.

**Comment by jmikrut** [comment]
Hey all, we are going to close this issue as we have resolved it for 2.0.0 - it's related to the work we're shipping in database adapters.
